opt/jdbc.sh

```diff
@@ -8,7 +8,12 @@ set_jdbc_url() {
       local db_protocol=$(expr "$db_url" : "\(.\+\)://")
       if [ "$db_protocol" = "postgres" ]; then
         local jdbc_protocol="jdbc:postgresql"
-        local db_default_args="&sslmode=require"
+
+        if [ "$CI" != "true" ]; then
```

Comment:
Does this not potentially cause failures when used in a context with `set -e`?

Reply:
true, we don't `set -e` in profile.d, but someone could and it would bleed over right?